<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>notepadClass</class>
 <widget class="QMainWindow" name="notepadClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>notepad</string>
  </property>
  <property name="windowIcon">
   <iconset resource="notepad.qrc">
    <normaloff>:/image/images/qt32.ico</normaloff>:/image/images/qt32.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>10</y>
      <width>571</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>596</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actNew"/>
    <addaction name="separator"/>
    <addaction name="actOpen"/>
    <addaction name="actClose"/>
    <addaction name="separator"/>
    <addaction name="actSave"/>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="actCopy"/>
    <addaction name="actCut"/>
    <addaction name="actPaste"/>
    <addaction name="actSelectAll"/>
   </widget>
   <widget class="QMenu" name="menu_style">
    <property name="title">
     <string>格式(&amp;S)</string>
    </property>
    <addaction name="actBold"/>
    <addaction name="actItalic"/>
    <addaction name="actUnderline"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_edit"/>
   <addaction name="menu_style"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar_file">
   <property name="windowTitle">
    <string>文件</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actNew"/>
   <addaction name="actOpen"/>
   <addaction name="actClose"/>
   <addaction name="actSave"/>
   <addaction name="actSaveAs"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar_edit">
   <property name="windowTitle">
    <string>编辑</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actCopy"/>
   <addaction name="actCut"/>
   <addaction name="actPaste"/>
  </widget>
  <widget class="QToolBar" name="toolBar_style">
   <property name="windowTitle">
    <string>格式</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actBold"/>
   <addaction name="actItalic"/>
   <addaction name="actUnderline"/>
  </widget>
  <action name="actOpen">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/image/images/open.png</normaloff>
     <normalon>:/image/images/new.png</normalon>:/image/images/open.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actCopy">
   <property name="icon">
    <iconset>
     <normalon>:/image/images/copy.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="toolTip">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actCut">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/image/images/cut.png</normaloff>:/image/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="toolTip">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actPaste">
   <property name="icon">
    <iconset>
     <normalon>:/image/images/paste.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="toolTip">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actSelectAll">
   <property name="icon">
    <iconset>
     <normalon>:/image/images/open.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>全选</string>
   </property>
   <property name="toolTip">
    <string>全选</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/image/images/textbold.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>粗体</string>
   </property>
   <property name="toolTip">
    <string>加粗字体</string>
   </property>
  </action>
  <action name="actSave">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/image/images/filesave.png</normaloff>:/image/images/filesave.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actItalic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/image/images/textitalic.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>斜体</string>
   </property>
   <property name="toolTip">
    <string>斜体</string>
   </property>
  </action>
  <action name="actUnderline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/image/images/textunder.png</normaloff>:/image/images/textunder.png</iconset>
   </property>
   <property name="text">
    <string>下划线</string>
   </property>
   <property name="toolTip">
    <string>添加下划线</string>
   </property>
  </action>
  <action name="actClose">
   <property name="icon">
    <iconset>
     <normalon>:/image/images/fileopen.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="toolTip">
    <string>关闭文件</string>
   </property>
  </action>
  <action name="actNew">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/image/images/new.png</normaloff>:/image/images/new.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actSaveAs">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/image/images/exportpdf.png</normaloff>:/image/images/exportpdf.png</iconset>
   </property>
   <property name="text">
    <string>另存为</string>
   </property>
   <property name="toolTip">
    <string>另存为</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="notepad.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actCut</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actPaste</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actCopy</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actItalic</sender>
   <signal>triggered(bool)</signal>
   <receiver>textEdit</receiver>
   <slot>setFontItalic(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actUnderline</sender>
   <signal>triggered(bool)</signal>
   <receiver>textEdit</receiver>
   <slot>setFontUnderline(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
